src/NodeAnalyzer/ParamAnalyzer.php

```diff
@@ -164,4 +168,17 @@ private function isParamUsed(Node $node, Param $param): bool
         $arguments = $this->funcCallManipulator->extractArgumentsFromCompactFuncCalls([$node]);
         return $this->nodeNameResolver->isNames($param, $arguments);
     }
+
+    private function isFuncGetArgsCall(Node $node): bool
```

Comment:
this should can use `VariadicFunctionLikeDetector::isVariadic()` that cover 3 functions:

- func_get_arg
- func_get_args
- func_num_args

https://github.com/rectorphp/rector-src/blob/dc677e1b15aaa2015959d62b3db812aecb4dedf1/rules/DeadCode/NodeManipulator/VariadicFunctionLikeDetector.php#L30

Reply:
I created PR for it:

- https://github.com/rectorphp/rector-src/pull/7016

to merge functionality into `ParamAnalyzer` and remove the `VariadicFunctionLikeDetector` as double work and cause manually called.